src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const formatedCSSData = {};
+  const splitedSource = sourceString.split(';');
```

Comment:
The variable name 'splitedSource' contains a typo and is not descriptive. Consider renaming it to 'splitSource' or 'styleDeclarations' to both correct the typo and improve clarity.